src/Compilers/Core/CodeAnalysisTest/MetadataReferences/MetadataReferenceTests.cs

```diff
@@ -433,8 +434,8 @@ protected override PortableExecutableReference WithPropertiesImpl(MetadataRefere
 
         private class MyReference2 : PortableExecutableReference
         {
-            public MyReference2(string fullPath, string display)
-                : base(default(MetadataReferenceProperties), fullPath)
+            public MyReference2(string fullPath)
+                : base(default, fullPath)
```

Comment:
Agreed. Will add argument name. The IDE was showing the type as unnecessary and "Simplify" just removed the type name instead of adding a named argument. Filed https://github.com/dotnet/roslyn/issues/63238